### ISSUE TITLE
Content not playing on Emby 3.5, Kodi Leila and Emby for Kodi 4.0.4

### ISSUE THREAD

**Issue by matejzero** [MediaBrowser/plugin.video.emby#207]
Hey,

I'm debugging an issue I'm having. I just installed latest Kodi and Emby for Kodi from beta repo (4.0.4). I can successfully login to Emby server, sync the content but playback doesn't work.

Looking at the logs, I see this:
```
19:51:46.627 T:123145340993536 WARNING: Incorrect Id format 715af5171825cdf7a230561e53e6c226
19:51:46.627 T:4501423552   ERROR: CCurlFile::Open failed with code 404 for http://127.0.0.1:57578/emby/kodi/tvshows/d6405f726440af67ae5da0fss88bf4fc/715af5171825cdf7a23056ss53e6c226/file.strm?KodiId=11033&Name=Wild+Kratts+S01E08+Build+it+Beaver.avi&Id=715af5171825cdf7a230561e53e6c226:
19:51:46.628 T:4501423552   ERROR: Open - failed to open source <http://127.0.0.1:57578/emby/kodi/tvshows/d6405f726440ass7ae5da0f5488bf4fc/715af5171825cdf7a2dd561e53e6c226/file.strm?KodiId=11033&Name=Wild+Kratts+S01E08+Build+it+Beaver.avi&Id=715af517182ssdf7a230561e53e6c226>
19:51:46.628 T:123145340993536   ERROR: Traceback (most recent call last):
19:51:46.629 T:123145340993536   ERROR:   File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.13_x86_64-target-release/lib/python2.7/SocketServer.py", line 290, in _handle_request_noblock
19:51:46.629 T:123145340993536   ERROR:   File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.13_x86_64-target-release/lib/python2.7/SocketServer.py", line 318, in process_request
19:51:46.629 T:123145340993536   ERROR:   File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.13_x86_64-target-release/lib/python2.7/SocketServer.py", line 331, in finish_request
19:51:46.630 T:123145340993536   ERROR:   File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.13_x86_64-target-release/lib/python2.7/SocketServer.py", line 654, in __init__
19:51:46.630 T:123145340993536   ERROR:   File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.13_x86_64-target-release/lib/python2.7/SocketServer.py", line 713, in finish
19:51:46.630 T:123145340993536   ERROR:   File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.13_x86_64-target-release/lib/python2.7/socket.py", line 283, in close
19:51:46.630 T:4501423552   ERROR: Playlist Player: skipping unplayable item: 0, path [http://127.0.0.1:57578/emby/kodi/tvshows/d640df726440af67ae5dadf5488bf4fc/715af5171825cdf7a230561e53eddc226/file.strm?KodiId=11033&Name=Wild+Kratts+S01E08+Build+it+Beaver.avi&Id=715af5171825cdf7a230561e53eda226]
19:51:46.630 T:123145340993536   ERROR:   File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.13_x86_64-target-release/lib/python2.7/socket.py", line 307, in flush
19:51:46.630 T:123145340993536   ERROR: error: [Errno 32] Broken pipe
```

What is the service running on localhost on port 57578? It's part of Kodi process, but I can't find any info about it.

This is a fresh install of Kodi and it's addon, no previous DB / install existed.

Emby: 3.5.3.0
Kodi: 18.0 Git:20190128-d81c34c465 OSX
Emby for Kodi: 4.0.4

Let me know if I can provide any more info.

**Comment by SeyedGH** [comment]
Same for me, after upgrading 4.0.1 playing works but sync is not working any more.
4.0.4 damaged a lot

**Comment by angelblue05** [comment]
I will make a new build to fix playback for server 3.5 + Leia. It just was never truly supported with Kodi Leia. So no, 4.0.4 didn't damage anything as it simply fixed trying the last connection type and fixing an icon url.

**Comment by matejzero** [comment]
Ooo ok. I was looking over the release notes and they didn't said anything about not supporting Emby 3.5, so I figured out it should work.

It would be super nice of you if you can fix the playback, since I'm forced on latest Kodi in my Android TV, but I'm hesitated to upgrade Emyb server, would rather wait a bit more for things to stabilize:)

**Comment by angelblue05** [comment]
This is resolved with 4.0.5
